test/helpers/contracts.js

```diff
@@ -9,6 +10,27 @@ function getLogs(Event, filter, additionalFilters) {
   return promisify(query.get.bind(query))();
 }
 
+const deployOrGetRootRegistry = async config => {
+  const { network, artifacts, deployer } = config;
```

Comment:
You can put this object destructuring directly in the function parameter declaration and it will serve as documentation.